changelogs/fragments/comment_fail.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+   - comment - raise exception when an invalid value provided to comment filter.
```

An exception was already raised before (KeyError). Instead, this PR improves the error messaging:

```suggestion
   - comment filter - Improve the error message shown when an invalid ``style`` argument is provided.
```